payload/reggae/options.d

```diff
@@ -57,12 +57,7 @@ struct Options {
     string dubObjsDir;
     bool dubDepObjsInsteadOfStaticLib;
     string dubBuildType = "debug";
-
-
-    version(Windows)
-        DubArchitecture dubArch = DubArchitecture.x86_mscoff;
-    else
-        DubArchitecture dubArch = DubArchitecture.x86_64;
+    string dubArchOverride;
```

Comment:
Why remove the default?

Reply:
This was only used when invoking dub, not for the lib interop at all. Getting rid of it means we invoke dub without any `--arch`, which is in line with letting dub figure out the default arch later on.